### ISSUE TITLE
FormatException when starting the game

### ISSUE THREAD

**Issue by nikita02722** [anegostudios/VintageStory-Issues#662]
The game is licensed, after entering the email and password, the crash occurs

```
Running on 64 bit Windows with 16 GB RAM 
Version: v1.13.4 (Stable)25.11.2020 17:00:09: Critical error occurred
System.FormatException: Входная строка имела неверный формат.
   в System.Text.StringBuilder.FormatError()
   в System.Text.StringBuilder.AppendFormatHelper(IFormatProvider provider, String format, ParamsArray args)
   в System.String.FormatHelper(IFormatProvider provider, String format, ParamsArray args)
   в System.String.Format(IFormatProvider provider, String format, Object[] args)
   в System.IO.TextWriter.WriteLine(String format, Object[] arg)
   в System.IO.TextWriter.SyncTextWriter.WriteLine(String format, Object[] arg)
   в System.Console.WriteLine(String format, Object[] arg)
   в _6Ki7onGUDN2hMDvAS3I5KRRsLrr.Log(EnumLogType logType, String message, Object[] args)
   в Vintagestory.Common.ModLoader.CollectMods()
   в Vintagestory.Common.ModLoader.LoadModInfos()
   в _LSAAgZhw31Rqs2is6nnojfZBbQC._MvWOQXBvCJeGuNEdbyOfgtCPh3P()
   в _LSAAgZhw31Rqs2is6nnojfZBbQC._HBADFZBlHRfVx9HpAQjhFIo25xm()
   в System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   в System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   в System.Threading.QueueUserWorkItemCallback.System.Threading.IThreadPoolWorkItem.ExecuteWorkItem()
   в System.Threading.ThreadPoolWorkQueue.Dispatch()
-------------------------------
```

**Comment by copygirl** [comment]
This appears to occur when the game is loading mods. Can you give me a list of the mods you have installed along with their filenames? (I have a hunch what it might be.)

**Comment by Yvan2222** [comment]
hello, I have the same fashion error, I removed it from the Mods folder, but the problem did not go away, I write this
Running on 32 bit Windows with 4 GB RAM
Version: v1.13.4 (Stable) artEA # 800126.11.2020 15:12:34: Critical error occurred
System.MissingMethodException: Method not found: 'Void System.Runtime.GCSettings.set_LargeObjectHeapCompactionMode (System.Runtime.GCLargeObjectHeapCompactionMode)'.
   at Vintagestory.Client.NoObf.ClientPlatformWindows..ctor (_6Ki7onGUDN2hMDvAS3I5KRRsLrr logger)
   at _lYLQLGyldvbDG9rPQyiRVeIqQVp._EEzpApWBQHtHR4iMJabXK2wDvlA (_ex0VSgRAdtsvhANmJHZHaf9ATWt, String [])
   at _lYLQLGyldvbDG9rPQyiRVeIqQVp._kpmnXdlXbBpeuPnzZqsMVNtiY3p._lPFPqQABMIMe0xhHIiuKuwpai51 ()
   at _IcGByoJhapvESnm6vaOawAHbkeU._EEzpApWBQHtHR4iMJabXK2wDvlA (ThreadStart)
-------------------------------
help me please

**Comment by Yvan2222** [comment]
I have a game without a license
the site from which I downloaded it, tested by experience, it is normal without viruses

**Comment by copygirl** [comment]
@Yvan2222 This is a different crash. Please see [this issue](https://github.com/anegostudios/VintageStory-Issues/issues/591).
(A bit odd for you to be using an official support channel with a pirated copy of the game.)

**Comment by Yvan2222** [comment]
it didn't help me it appeared
Running on 32 bit Windows with 4 GB RAM 
Version: v1.13.4 (Стабильно) artEA#800126.11.2020 17:43:05: Critical error occurred
System.NotSupportedException: Your graphics card supports only OpenGL version 140 (1.40 - Intel Build 9.17.10.4229), but OpenGL version 330 is required.
Please check if you have installed the latest version of your graphics card driver. If they are, your graphics card may be to old to play Vintage Story.(Note: In case of modded gameplay with modded shaders, the mod author may be able to lower the OpenGL version requirements)
   at Vintagestory.Client.NoObf.Shader.EnsureVersionSupported(String versionUsed, String ownFilename)
   at Vintagestory.Client.NoObf.Shader.EnsureVersionSupported()
   at Vintagestory.Client.NoObf.ShaderProgram.Compile()
   at Vintagestory.Client.NoObf.ShaderRegistry.loadRegisteredShaderPrograms()
   at Vintagestory.Client.NoObf.ShaderRegistry.Load()
   at _LSAAgZhw31Rqs2is6nnojfZBbQC._BL0PigheDieo6G01vpqOQ8xL3Od()
   at _2CEJrix6eZjnKir3XZC1241baqf._toICAyZ9EYSaLsmD4KTuQWUbqjP(Single )
   at _LSAAgZhw31Rqs2is6nnojfZBbQC._4BykKOCicJqCBHpyWIgTmxWrTrg(Single )
   at _LSAAgZhw31Rqs2is6nnojfZBbQC._6XmOxKoTYrFLRCGd58p58XW43nD(Single )
   at Vintagestory.Client.NoObf.ClientPlatformWindows.window_RenderFrame(Object sender, FrameEventArgs e)
   at System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   at OpenTK.GameWindow.OnRenderFrame(FrameEventArgs e)
   at OpenTK.GameWindow.RaiseRenderFrame(Double elapsed, Double& timestamp)
   at OpenTK.GameWindow.DispatchRenderFrame()
   at OpenTK.GameWindow.Run(Double updates_per_second, Double frames_per_second)
   at _lYLQLGyldvbDG9rPQyiRVeIqQVp._EEzpApWBQHtHR4iMJabXK2wDvlA(_ex0VSgRAdtsvhANmJHZHaf9ATWt , String[] )
   at _lYLQLGyldvbDG9rPQyiRVeIqQVp._kpmnXdlXbBpeuPnzZqsMVNtiY3p._lPFPqQABMIMe0xhHIiuKuwpai51()
   at _IcGByoJhapvESnm6vaOawAHbkeU._EEzpApWBQHtHR4iMJabXK2wDvlA(ThreadStart )
-------------------------------

**Comment by copygirl** [comment]
@Yvan2222 This is not a support forum. You're now responding to an issue that is unrelated. In the future, please try to search for existing issues with the same crash. In this case, you're getting the same error as in #643. See my response there.

**Comment by copygirl** [comment]
As for the original crash that @nikita02722 is getting: I provided a fix for it to the developers, which will hopefully be included in an upcoming 1.14 release candidate. The issue is that the install or user folder contains "formatting symbols", perhaps `{` or `}`.

**Comment by copygirl** [comment]
Will be fixed in `1.14.0-rc.5` and later.
